## rename-genbox-src.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: rename genbox in src

@DPATCH@
--- a/src/gen/Rmakefile
+++ b/src/gen/Rmakefile
@@ -13,7 +13,7 @@ INSTDIR = /usr/local/bin
 
 LIBDIR = /usr/local/lib/ray
 
-PROGS = genbox gensky xform genblinds genrev genworm gendaylit \
+PROGS = genrbox gensky xform genblinds genrev genworm gendaylit \
 		gensurf genprism replmarks mkillum mksource genclock
 
 LIBFILES = illum.cal rev.cal skybright.cal surf.cal clockface.hex \
@@ -34,8 +34,8 @@ clean:
 genbeads:	genbeads.o hermite3.o
 	$(CC) $(CFLAGS) -o genbeads genbeads.o hermite3.o $(MLIB)
 
-genbox:	genbox.o
-	$(CC) $(CFLAGS) -o genbox genbox.o
+genrbox:	genbox.o
+	$(CC) $(CFLAGS) -o genrbox genbox.o
 
 genmarble:	genmarble.o ../common/random.h
 	$(CC) $(CFLAGS) -o genmarble genmarble.o $(MLIB)
